pkg/collector/tree.go

```diff
@@ -271,6 +271,35 @@ func labelStringToLabels(input string) []label {
 	return res
 }
 
+func labelIdentifierToLabels(id identifier) []label {
```

Comment:
Can we maybe add a test for this function?

Reply:
Just added.